### DISCUSSION TITLE
Does anyone have examples of custom emitters in python?

### DISCUSSION THREAD

**Discussion by a3ahmad** [mitsuba-renderer/mitsuba3#524] [Q&A] [answered]
I’ve tried to make one, myself, but it’s reporting that sample_position isn’t implemented, but mine is… so, I’m confused. Was hoping to confirm my translation of c++ implementations to python was sound.

**Comment by njroussel** [comment, marked as answer]
Hi @a3ahmad 

It looks like we never added a Python binding for the `sample_position` method. Your code is most likely correct. I'll be pushing a fix soon.

**Reply by njroussel** [reply]
The fix is here:https://github.com/mitsuba-renderer/mitsuba3/commit/8f03c7db7b697a2bac17fe960a8d4a6863bece4d

This will be included in the next release, until then you'll need to build the project yourself.

**Reply by a3ahmad** [reply]
Thanks for the quick fix!
